src/components/LoginPage.jsx

```diff
@@ -0,0 +1,29 @@
+import React, { useState } from 'react';
+import axios from 'axios';
+
+const LoginPage = () => {
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+    try {
+      const response = await axios.post('/api/login', { email, password });
+      if (response.status === 200) {
```

Comment:
Directly manipulating the window.location.href for navigation is not recommended in React. Consider using react-router for navigation.